siliconcompiler/flows/asicflow.py

```diff
@@ -64,7 +64,6 @@ def setup(chip, flowname='asicflow', syn_np=1, floorplan_np=1, physyn_np=1, plac
 
     #step -->(tool, task)
     tools = {
-        'import' : ['surelog', 'import'],
```

Comment:
Does the initial task get set up automatically from having `('option', 'frontend') == 'verilog'`?

Reply:
it does by the call to `flowtools = setup_frontend(chip)`